cmd/ftl/cmd_dev.go

```diff
@@ -105,10 +110,11 @@ func (d *devCmd) Run(ctx context.Context, client ftlv1connect.ControllerServiceC
 					modules.RemoveModule(dir)
 					// Increase delay when there's a compile failure.
 					delay = d.FailureDelay
+				} else {
+					currentModule.fileHashes = hashes
+					modules.SetModule(dir, currentModule)
```

Comment:
Ah thanks! I had this code in the `else` originally and have no idea why I decided to move it 🤦